libs/wingsdk/test/core/to-inflight-type.test.ts

```diff
@@ -0,0 +1,46 @@
+import * as vm from "vm";
+import { test, assert } from "vitest";
+import * as std from "../../src/std";
+import * as util from "../../src/util";
+
+const skip = [
```

Comment:
Should these be TODOs of some kind? (not sure)

Reply:
Mmm. not sure either. For now I think we are find because none of these have any static methods.